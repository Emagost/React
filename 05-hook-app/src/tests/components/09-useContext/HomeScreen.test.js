import { HomeScreen } from "../../../components/09-useContext/HomeScreen";
import UserContext from "../../../components/09-useContext/UserContext";
import {mount} from 'enzyme';

describe('test <HomeScreen/>', () => {

    const user = {
        name:'Emanuel',
        email: 'emanuel@react.com'
    }

    const wrapper = mount( 
        <UserContext.Provider value={{
            user
        }}>
            <HomeScreen/>
        </UserContext.Provider>
    );
    
    test('snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
