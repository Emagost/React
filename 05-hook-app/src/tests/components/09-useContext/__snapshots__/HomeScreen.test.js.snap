// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test <HomeScreen/> snapshot 1`] = `
Array [
  <h1>
    HomeScreen
  </h1>,
  <hr />,
  <pre>
    {
   "name": "Emanuel",
   "email": "emanuel@react.com"
}
  </pre>,
]
`;
